app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -308,6 +308,7 @@ def save_container_images_inventory(ems, hashes, target = nil)
 
     hashes.each do |h|
       h[:container_image_registry_id] = h[:container_image_registry][:id] unless h[:container_image_registry].nil?
+      h[:type] = h[:ems_ref] ? 'OpenshiftContainerImage' : 'BasicContainerImage'
```

Comment:
We cannot have provider specifics here.

This can possibly be done differently by setting the :type in the parser itself.  Then here do `h[:type] ||= 'BasicContainerImage'`.  Maybe?  I'm not exactly sure what a BasicContainerImage even is and whether or not it belongs in core or in the OpenShift specific provider code.